### DISCUSSION TITLE
[FR] Make use of relational fields more straightforward (less clicks/keyboard strokes)

### DISCUSSION THREAD

**Discussion by wsydney76** [craftcms/cms#8891] [Ideas]
### Description

Opening a modal, finding, selecting an element each time can be a bit inconvinient, especially when you have to manage a lot of content with a lot of fields in a short time. For example think of a big film festival with hundreds of movies/sections/person profiles/screenings/reviews. For each movie select section/genre/country/language/director/producer/cast/credits/awards won etc.

Most of the time you know the title of the related element, or if it will be a new one.

So instead of having a single button to open the modal, this is the clients wish list:

a) have a text field where you can enter the title, and select from suggestions (like the tag field)
b) have a "Select" button, opening the modal
c) have a "Add new" button, opening the HUD editor
d) special bonus: have a list of most /most recently used elements to select from

e) make everything keyboard accessible

**Comment by piotrpog** [comment]
This is a good idea for plugin actually. One that would overwrite default relation fields with what you describe.

**Comment by sjelfull** [comment]
The modal window can also be kind of slow to open/close due to animations + load time, making the whole workflow a bit....... zzzzzzzzlow.
